test-echo/README.md

```diff
@@ -40,10 +40,10 @@ image.save('/envoyai/output/image.png','PNG')
 
 ## 2 Input
 
-In practice, all inputs defined by your algorithms `com.envoyai.schema-in` will be supplied via JSON over WebAPI via the 
-'EnvoyAI Integration API', and one should not be too concerned with the exact user interface, as it may need to vary 
-system to system, or client to client. However for testing and demonstration purposes, at https://portal.envoyai.com 
-we dynamically render input controls for any input type.
+In practice, all inputs defined by your algorithm `com.envoyai.schema-in` will be supplied via JSON over HTTP/HTTPS via the 
+'EnvoyAI Integration API'.  One should not be concerned with the exact user interface presentation/experience, as it may need to vary 

```

Comment:
I don't like experience here

Reply:
Fixed as well as the HTTP/HTTPS -> WebApi